mlflow/pyfunc/backend.py

```diff
@@ -91,11 +92,11 @@ def __init__(
         """
         super().__init__(config=config, **kwargs)
         self._nworkers = workers or 1
-        if env_manager == _EnvManager.CONDA and ENV not in config:
+        if env_manager == CONDA and ENV not in config:
             warnings.warn(
                 "Conda environment is not specified in config `env`. Using local environment."
             )
-            env_manager = _EnvManager.LOCAL
+            env_manager = LOCAL
```

Comment:
nit:
I prefer `_EnvManager.LOCAL` way, `LOCAL` is a short word and using namespace `_EnvManager` avoids potential name conflicts. Similar to CONDA ,  VIRTUALENV

Reply:
I do see your point. My focus here is on readability and compactness. I see in other parts of the codebase we use `em` instead of `_EnvManager` to represent the actual module `env_manager`. 

Could we consider renaming `LOCAL`  `CONDA` to `LOCALENV` and `CONDAENV` ?

Reply:
then I think `em.LOCAL` is a better way

Reply:
ok, agreed. done!